c++/src/capnp/rpc.c++

```diff
@@ -112,11 +112,16 @@ Orphan<List<rpc::PromisedAnswer::Op>> fromPipelineOps(
 }
 
 kj::Exception toException(const rpc::Exception::Reader& exception) {
-  return kj::Exception(static_cast<kj::Exception::Type>(exception.getType()),
+  kj::Exception result(static_cast<kj::Exception::Type>(exception.getType()),
       "(remote)", 0, kj::str("remote exception: ", exception.getReason()));
+  if (exception.hasTrace()) {
+    result.setRemoteTrace(kj::str(exception.getTrace()));
+  }
+  return result;
 }
 
-void fromException(const kj::Exception& exception, rpc::Exception::Builder builder) {
+void fromException(const kj::Exception& exception, rpc::Exception::Builder builder,
+                   kj::Maybe<kj::Function<kj::String(const kj::Exception&)>&> traceEncoder) {
   // TODO(someday):  Indicate the remote server name as part of the stack trace.  Maybe even
```

Comment:
Looks like you can knock off the "Maybe even transmit stack traces" part of this TODO